src/fe/fe_bernstein.C

```diff
@@ -97,8 +96,7 @@ void bernstein_nodal_soln(const Elem * elem,
             // u_i = Sum (alpha_i phi_i)
             for (unsigned int i=0; i<n_sf; i++)
               nodal_soln[n] += elem_soln[i] *
-                // FE<Dim,T>::shape(elem, order, i, mapped_point);
-                FEInterface::shape(Dim, fe_type, elem, i, refspace_nodes[n]);
+                FEInterface::shape(Dim, p_refined_fe_type, elem, i, refspace_nodes[n]);
```

Comment:
The switch from fe_type to p_refined_fe_type here - was that a bug in the previous code?

Reply:
No, that's a temporary workaround until I have a chance to update `FEInterface::shape()` so that it handles Elem::p_level() internally. That should be my next PR.